libqfieldsync/offline_converter.py

```diff
@@ -266,14 +268,16 @@ def _convert(self, project: QgsProject) -> None:
             elif (
                 layer_action == SyncAction.COPY or layer_action == SyncAction.NO_ACTION
             ):
-                copied_files = layer_source.copy(self.export_folder, copied_files)
+                copied_files = layer_source.copy(
+                    self._export_filename.parent, copied_files
+                )
             elif layer_action == SyncAction.KEEP_EXISTENT:
-                layer_source.copy(self.export_folder, copied_files, True)
+                layer_source.copy(self._export_filename.parent, copied_files, True)
             elif layer_action == SyncAction.REMOVE:
                 project.removeMapLayer(layer)
 
-        export_project_filename = self.export_folder.joinpath(
-            f"{self.original_filename.stem}_qfield.qgs"
+        export_project_filename = self._export_filename.parent.joinpath(
+            f"{self._export_filename.stem}.qgs"
         )
```

Comment:
```suggestion
        export_project_filename = self._export_filename
```

This is enough, we can add a check earlier that `self._export_filename` is a `.qgs` file.

This should also be handled in the UI.